src/vignette/http/middleware.clj

```diff
@@ -21,12 +21,14 @@
               thumb-map (:thumb-map e) ; if present, we'll try to thumbnail the error response
               response-code (or (:response-code e) 500)
               context (assoc (dissoc e :type :thumb-map :response-code) :host hostname)]
-          (perf/publish {:convert-error 1})
-          (println message ":" (:uri request))
-          (log/warn message
-                    (merge {:path (:uri request)
-                            :query (:query-string request)}
-                           context))
+          (when (>= response-code 500)
```

Comment:
Hmm. This looked odd at first. Will we ever get something other than a `5xx` here?